ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -24,112 +25,57 @@ def __init__(self, driver):
-        self.mode = mode
-
-    @property
-    def select_storage_class(self):
-        return self.storage_class_type
-
-    @select_storage_class.setter
-    def select_storage_class(self, storage_class):
-        if not isinstance(storage_class, str):
-            raise ValueError("storage class arg must be a string")
-        self.storage_class = storage_class
-
-    @property
-    def select_osd_size(self):
-        return self.osd_size
-
-    @select_osd_size.setter
-    def select_osd_size(self, osd_size):
-        if not isinstance(osd_size, str):
-            raise ValueError("osd size arg must be a string")
-        self.osd_size = osd_size
-
-    @property
-    def wide_encryption(self):
-        return self.is_wide_encryption
-
-    @wide_encryption.setter
-    def wide_encryption(self, is_wide_encryption):
-        if not isinstance(is_wide_encryption, bool):
-            raise ValueError("is_wide_encryption arg must be a bool")
-        self.is_wide_encryption = is_wide_encryption
-
-    @property
-    def class_encryption(self):
-        return self.is_class_encryption
-
-    @class_encryption.setter
-    def class_encryption(self, is_class_encryption):
-        if not isinstance(is_class_encryption, bool):
-            raise ValueError("is_class_encryption arg must be a bool")
-        self.is_class_encryption = is_class_encryption
-
-    @property
-    def select_service_name(self):
-        return self.service_name
-
-    @select_service_name.setter
-    def select_service_name(self, service_name):
-        if not isinstance(service_name, str):
-            raise ValueError("service_name arg must be a string")
-        self.service_name = service_name
-
-    @property
-    def use_kms(self):
-        return self.is_use_kms
-
-    @use_kms.setter
-    def use_kms(self, is_use_kms):
-        if not isinstance(is_use_kms, bool):
-            raise ValueError("is_use_kms arg must be a bool")
-        self.is_use_kms = is_use_kms
-
-    @property
-    def select_kms_address(self):
-        return self.kms_address
-
-    @select_kms_address.setter
-    def select_kms_address(self, kms_address):
-        if not isinstance(kms_address, str):
-            raise ValueError("kms_address arg must be a string")
-        self.kms_address = kms_address
-
-    @property
-    def select_kms_address_port(self):
-        return self.kms_address_port
-
-    @select_kms_address_port.setter
-    def select_kms_address_port(self, kms_address_port):
-        if not isinstance(kms_address_port, str):
-            raise ValueError("kms_address_port arg must be a string")
-        self.kms_address_port = kms_address_port
-
-    @property
-    def select_kms_token(self):
-        return self.kms_token
-
-    @select_kms_token.setter
-    def select_kms_token(self, kms_token):
-        if not isinstance(kms_token, str):
-            raise ValueError("kms_token arg must be a string")
-        self.kms_token = kms_token
+
+    def add_disks_vmware_lso(self):
```

Comment:
I don't think you need to have this function implemented here. 
1) it is not specific for UI testing
2) it is specific for vsphere, while we already have https://github.com/red-hat-storage/ocs-ci/blob/8dcae7014a94abfc18ef8538f17c94bd98f31f2c/ocs_ci/deployment/deployment.py#L476 which is generic 

Please try to use the call to setup_local_storage() by doing the UI installation after it is called in deploy_ocs_via_operator()

Reply:
I think that here we don't need all that is written in the function `setup_local_storage`. We need only the part that Oded added here. However, there is no point in writing a code twice. So my suggestion is to add a new function in the file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/helpers/lso_helpers.py - where the function `setup_local_storage` is located. We can call it something like `add_disk_for_vsphere_platform()` and copy-paste the code above to this function. Then we can use it for both in `setup_local_storage` and here.

Reply:
done

Reply:
do we still need this function? can't we directly call `add_disk_for_vsphere_platform()` from whatever needed under the same condition of LSO?